src/components/06-miscellaneous/03-toolbar/toolbar.njk

```diff
@@ -1,256 +1,135 @@
-                      </button>
-                     
-                    <div id="accordion-serv-class-content" role="region" aria-labelledby="accordion-serv-class-id" class="sds-accordion__panel" hidden="">
-                      <div class="sds-accordion__panel-body">
-                        <label class="usa-label" for="input-type-naics">NAICS</label >
-                        <input class="usa-input" id="input-type-naics" name="input-type-naics" type="text"/>
-                        <div class="selected-chips margin-top-05">
-                          <span class="sds-tag sds-tag--chip">
-                            111140 -- Wheat Farming
-                            <button class="sds-tag__close">
-                              <span class="fas fa-times" aria-hidden="true"></span>
-                            </button>
-                          </span>
-                          <span class="sds-tag sds-tag--chip">
-                            212222 -- Silver Ore Mining
-                            <button class="sds-tag__close">
-                              <span class="fas fa-times" aria-hidden="true"></span>
-                            </button>
-                          </span>
-                          <span class="sds-tag sds-tag--chip">
-                            541850 -- Outdoor Advertising
-                            <button class="sds-tag__close">
-                              <span class="fas fa-times" aria-hidden="true"></span>
-                            </button>
-                          </span>
-                        </div>
-                        <label class="usa-label" for="input-type-psc">PSC</label>
-                        <input class="usa-input" id="input-type-psc" name="input-type-psc" type="text"/>
-                      </div>
-                    </div>
-                  </div>
-                  <div class="sds-accordion__item">
-                     
-                      <button aria-expanded="false" class="sds-accordion__trigger" aria-controls="accordion-entity-content" id="accordion-entity-id">
-                        <span class="sds-accordion__title" role="heading">
-                          Entity Name/UEI
-                          <span class="sds-accordion__icon"></span>
-                        </span>
-                      </button>
-                     
-                    <div id="accordion-entity-content" role="region" aria-labelledby="accordion-entity-id" class="sds-accordion__panel" hidden="">
-                      <div class="sds-accordion__panel-body">
-                        <label class="usa-label" for="input-type-entity">Name</label >
-                        <input class="usa-input" id="input-type-entity" name="input-type-entity" type="text"/>
-                      </div>
+        </div>
+      </div>
+      <div class="sds-accordion__item">
+
+        <button aria-expanded="false" class="sds-accordion__trigger" aria-controls="accordion-filters-content"
```

Comment:
are accordions supposed to not work in preview?

Reply:
accordions needs some work

Reply:
a separate story is required to address this